native/src/utils/DatabaseConnector.ts

```diff
@@ -314,10 +314,10 @@ class DatabaseConnector {
         }): {
           last_update: string
         } => ({
-          last_update: lastUpdate.toISOString(),
+          last_update: lastUpdate.toJSDate().toISOString(),
```

Comment:
@steffenkleinle I can't change it just ** to toISO** because it giving off an error saying `TS2322: Type 'string | null' is not assignable to type 'string'.   Type 'null' is not assignable to type 'string'.`

Reply:
Hmm yeah: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3420adf4ca802933d360aced242b4a44ebca507d/types/luxon/src/duration.d.ts#L269

Does anybody have an idea how we should handle this? I think it is super annoying that we always have to take care of possibly invalid dates and therefore null values... Could we just override the types as we should not have invalid dates (except the cms fucks it up)? Perhaps @sarahsporck?

Reply:
replaced `toJSDate().toISOString()` to `toISO()` without any problem :+1: